x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/pipeline_processors_editor_item/inline_text_input.tsx

```diff
@@ -89,6 +89,7 @@ function _InlineTextInput({
       className={containerClasses}
       tabIndex={disabled ? -1 : 0}
       onFocus={() => setIsShowingTextInput(true)}
+      title={text ?? placeholder}
```

Comment:
You don't need to avoid it per say... But you also can't assume that it does anything... 

So if you're trying to communicate anything to screen readers, you shouldn't assume that `title` will work.

We also can't use `aria-label` or any of its related attributes here because on a `div` with no `role`, that won't work either.

So if you're looking to provide more context specifically to screen readers, visually hidden text is the way to go which you can do by wrapping some text in `<EuiScreenreaderOnly>`

Reply:
Thanks both for the response! The intention here is to provide a way for users to still see the full text even when it is truncated.

`EuiTooltip` sounds like the way to go :+1:. @myasonik , if the div that contains the descriptive text turns into an `EuiTextField` on focus is it perhaps not worth adding the `EuiScreenReaderOnly`? Further context https://github.com/elastic/eui/issues/3928 is the behaviour of the text.

Reply:
Yup, I don't think you need to add anything other than `EuiTooltip` 👍

Reply:
This is how the case with truncated text is handled now

<img width="1246" alt="Screenshot 2021-04-07 at 11 36 44" src="https://user-images.githubusercontent.com/8155004/113845319-ac34e100-9795-11eb-815e-6f35e037062e.png">